### ISSUE TITLE
FormData [ReferenceError: FormData is not defined]

### ISSUE THREAD

**Issue by mischizzle** [tmpvar/jsdom#1551]
As part of a suite of tests in a mocha environment, FormData is now not defined. This is making the tests fail. Could be perhaps part of the 9.4 updates. 

`ReferenceError: FormData is not defined`

Revering to the previous version fixes the issue
`  "jsdom": "9.3.0",`

**Comment by domenic** [comment]
Cannot reproduce:

```js
const jsdom = require("jsdom");
const window = jsdom.jsdom().defaultView;

// From inside:
window.eval(`console.log(FormData))`);

// From outside:
console.log(window.FormData);
```

both console.logs work.

Please provide a small test case, not using any external dependencies (like mocha), to show the problem.

**Comment by Sebmaster** [comment]
Probably a dupe of #1549.

**Comment by martijndeh** [comment]
It looks like `FormData` is not an enumerable property in `jsdom@9.4.0` anymore. Before, in `jsdom@9.3.0` and earlier, `FormData` was an enumerable property.

This doesn't break in Mocha specifically, I think, but in `jsdom-global` (which I assume is often used in combination with Mocha).

To reproduce:

`$ npm install jsdom@9.3.0`

```js
const jsdom = require('jsdom');

const window = jsdom.jsdom().defaultView;

console.log(Object.keys(window).filter((key) => key === 'FormData'));
```

This logs an array with one value `FormData`. In `9.4.0` and up this logs an empty array.

**Comment by domenic** [comment]
Ah, yeah, that was a bug fix in 9.4.0. Closing since this is not a jsdom bug, but happy to continue helping in the closed thread.

You may also enjoy https://github.com/tmpvar/jsdom/wiki/Don't-stuff-jsdom-globals-onto-the-Node-global .
